src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  color: $main-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: $white-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+
+  &__image {
+    margin-bottom: 40px;
+    display: flex;
+    justify-content: center;
+    img {
+      display: block;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    color: $secondary-color;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+
+  &__title {
+    font-size: 12px;
+    color: $secondary-color;
+  }
+
+  &__count {
+    font-size: 16px;
+    font-weight: 700;
+  }
+}
+
+.button {
+  width: 100%;
+  height: 40px;
+  background-color: $blue-color;
+  color: $white-color;
+  font-family: "Roboto", sans-serif;
```

Comment:
it should be inherited from the card, so redundant here
Also, if it's an element of the card it should be properly named, otherwise it's a BEM block and should live in a separate file

Just FYI :arrow_up: - you've successfully fixed all previous issues and generally code looks very good :+1: - Approved